helm-charts/fluentd-es/templates/fluentd-es.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
From our conversation the other day, if you run `helm create newchart` and look at the generated `Deployment` template (or look at [nginx-ingress](https://github.com/helm/charts/blob/master/stable/nginx-ingress/templates/controller-deployment.yaml#L11)) it uses the fullname template (defined in `_helpers.tpl`). We should use the same and keep it consistent for all resources.